qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -335,6 +335,48 @@ def test_rmdir_p_dir_with_file(self):
         self.mount_a.stat(self.dir_name)
 
 
+class TestRm(TestCephFSShell):
+    dir_name = "test_rm"
+
+    def dir_does_not_exists(self):
```

Comment:
```suggestion
    def assert_dir_does_not_exists(self):
```
Otherwise, It's not clear, from its name, what this method does.